geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LogsAreFullyRedactedAcceptanceTest.java

```diff
@@ -64,54 +65,49 @@
   @Rule
   public GfshRule gfsh = new GfshRule();
 
-  @BeforeClass
-  public static void addSecuritySystemProperties() {
-    System.setProperty(gfshSysEnvUsername, gfshSysEnvUsernameValue);
-    System.setProperty(gfshSysEnvPassword, gfshSysEnvPasswordValue);
-    System.setProperty(gfshSysEnvUsernamePost, gfshSysEnvUsernameValuePost);
-    System.setProperty(gfshSysEnvPasswordPost, gfshSysEnvPasswordValuePost);
-  }
+  public LogsAreFullyRedactedAcceptanceTest() {}
 
   @Before
-  public void createPropertyFiles() throws IOException {
+  public void createDirectoriesAndFiles() throws IOException {
     propertyFile = gfsh.getTemporaryFolder().newFile("geode.properties");
     securityPropertyFile = gfsh.getTemporaryFolder().newFile("security.properties");
 
     Properties properties = new Properties();
     properties.setProperty(LOG_LEVEL, "debug");
+    properties.setProperty(propertyFileUsername, propertyFileUsernameValue);
```

Comment:
some minor comments, and it's also a personal choice, but I always find it's easier to read and understand the tester's intent if we put plain text here instead of having to refer to the value of these constants on top of the class file. In the production code, it's a good practice to define the constants, but for tests, I find it easier to read just like this;
properties.setProperty("security-username", "user");